src/zcl_abapgit_transport_mass.clas.locals_imp.abap

```diff
@@ -0,0 +1,341 @@
+********************************************************************************
```

Comment:
please remove these first 23 commented lines, we currently only have this a few places and not for each include

Reply:
Done.